build/kill-plugin.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Why is this script needed? This seams to be a task for the server to take care of.